partiql-spi/src/main/kotlin/org/partiql/spi/catalog/Session.kt

```diff
@@ -46,11 +46,9 @@ public interface Session {
          * Returns a [Session] with only the "empty" catalog implementation.
          */
         @JvmStatic
-        public fun empty(): Session = object : Session {
-            override fun getIdentity(): String = "unknown"
-            override fun getCatalog(): String = "empty"
-            override fun getCatalogs(): Catalogs = Catalogs.empty()
-            override fun getNamespace(): Namespace = Namespace.empty()
+        public fun empty(): Session {
+            val catalog = Catalog.empty("empty")
+            return builder().catalog(catalog.getName()).catalogs(catalog).build()
```

Comment:
Since we now have a system catalog, then I don't think we need to add _any_ catalogs with the empty session. But this means you cannot create tables or anything until you create a catalog. Also this means `getCatalog()` for the session will be "$system". 

This was necessary before because we needed the super-chain, but adding an empty catalog to an empty is no longer needed with $system at the end of the path-chain.